src/app/layout.tsx

```diff
@@ -21,6 +22,7 @@ export default function RootLayout({
           <NextAuthProvider>
                 {children}
           </NextAuthProvider>
+          <AppBar />
```

Comment:
We probably should put it inside the auth provider

noticed the app bar is customizable per user